packages/ui/package.json

```diff
@@ -0,0 +1,119 @@
+{
+    "name": "@onlook/ui",
+    "description": "A ui library for Onlook",
+    "version": "0.0.0",
+    "private": true,
+    "sideEffects": false,
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/onlook-dev/onlook.git"
+    },
+    "files": [
+        "tokens.ts",
+        "tailwind.config.ts",
+        "postcss.config.js",
+        "globals.css"
+    ],
+    "scripts": {
+        "clean": "rm -rf .turbo node_modules",
+        "lint": "eslint --fix .",
+        "format": "prettier --write .",
+        "typecheck": "tsc --noEmit"
+    },
+    "keywords": [
+        "onlook",
+        "ui"
+    ],
+    "author": {
+        "name": "Onlook",
+        "email": "contact@onlook.dev"
+    },
+    "license": "Apache-2.0",
+    "homepage": "https://onlook.dev",
+    "devDependencies": {
+        "@onlook/utility": "workspace:*",
+        "@types/color-namer": "^1.3.3",
+        "autoprefixer": "^10.4.20",
+        "react": "^18.3.1",
+        "react-dom": "^18.3.1",
+        "typescript": "^5.5.4"
+    },
+    "exports": {
+        "./color-picker": "./src/components/color-picker/index.tsx",
```

Comment:
Concerned about developer experience on this. Especially with adding new ShadCN UI components. Is there a more intuitive API here such as a catch-all export?